### ISSUE TITLE
Creating negative data points

### ISSUE THREAD

**Issue by EasternCaveMan** [AlexanderKroll/ESP#14]
Hello,
I hope this message finds you well. I have a question regarding Section `6. Adding negative data points`subsection `(a)(iii) Creating negative data points for the test set:`.  I noticed that you generated negative data points for both phylogenetic and experimental evidences in the training set. However, for the test set, you only generated negative data points for experimental evidences. Could you kindly explain the rationale behind this decision?

I am looking forward to hear from you.
Thank you

Best,
